packages/lesswrong/components/users/UsersMenu.tsx

```diff
@@ -222,9 +263,46 @@ const UsersMenu = ({classes}: {
   const hasBookmarks = (currentUser?.bookmarkedPostsMetadata.length ?? 0) >= 1;
 
   const order: (keyof typeof items)[] = isFriendlyUI
-    ? ["newPost", "newShortform", "newQuestion", "newDialogue", "divider", "newEvent", "newSequence"]
+    ? ["newPost", "newShortform", "divider", "newEvent", "newDialogue", "newSequence"]
     : ["newShortform", "newPost", "newWikitag", "newEvent"];
 
+  const writeNewNode = isFriendlyUI ? (
+    <InteractionWrapper>
+      <LWTooltip
```

Comment:
If I hover over the menu item to view this tooltip, then move the mouse to a different menu item the tooltip doesn't close:

<img width="496" alt="Screenshot 2025-02-24 at 16 07 33" src="https://github.com/user-attachments/assets/2222f908-b956-43f4-b7a1-5a1e05dcca58" />

<img width="440" alt="Screenshot 2025-02-24 at 16 07 35" src="https://github.com/user-attachments/assets/57365672-53ce-4d07-bc5a-a408d01da390" />